jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -73,6 +73,15 @@ def should_export_dbias(bias_shape, query_shape, layout) -> bool:
     _, _, q_N, _ = query_shape
   return b_B == 1 and b_N == q_N
 
+def get_large_negative_number(dtype):
+  # temp WAR as cuDNN has a bug for subtraction between two large negative value
+  if dtype == jnp.bfloat16:
+    return jnp.asarray(-2 << 40, dtype=dtype)
+  elif dtype == jnp.float16:
+    return jnp.asarray(-2 << 14, dtype=dtype)
+  else:
+    raise ValueError("Unsupported dtype for inputs.")
```

Comment:
It looks like the previous version of this function accepted more dtypes. Why is it okay to only allow b?float16 here?

Reply:
cuDNN Flash attention only support these 2 data type. Fp8 flash attention is not integrated yet.